src/main/groovy/com/entagen/jenkins/JenkinsJobManager.groovy

```diff
@@ -64,7 +65,9 @@ class JenkinsJobManager {
         for(ConcreteJob missingJob in missingJobs) {
             println "Creating missing job: ${missingJob.jobName} from ${missingJob.templateJob.jobName}"
             jenkinsApi.cloneJobForBranch(missingJob, templateJobs)
-            jenkinsApi.enableJob(missingJob.jobName)
+            if (!isUnitTestOnly && !missingJob.jobName.contains("Rev.com-featurebranch-deploy-feature")) {
```

Comment:
If we use `isUnitTestOnly` then we will have to run the script a second time with `-DbranchNameRegex=unittest\/.+` however we could update the regex to support both unittest and feature and here we could just do a check for 
`if (!missingJob.jobName.contains("Rev.com-branch-deploy-unittest")) {`

Thoughts?